Gruntfile.js

```diff
@@ -218,8 +240,7 @@ module.exports = function(grunt) {
           example: eg,
           file: md,
           markdown: markdown.join("\n"),
-          breadboard: hasPng ? templates.img({ png: png }) : "",
-          fritzing: hasFzz ? templates.fritzing({ fzz: fzz }) : ""
+          breadboards: images.slice(0, -1)
```

Comment:
Just an extra newline. I was trying to make it generate as close to the previous versions as possible.